src/libraries/System.Security.Principal.Windows/src/System/Security/Principal/SID.cs

```diff
@@ -696,37 +661,13 @@ internal static byte Revision
         // prevent the caller from messing with the internal representation.
         //
 
-        internal byte[] BinaryForm
-        {
-            get
-            {
-                return _binaryForm;
-            }
-        }
+        internal byte[] BinaryForm => _binaryForm;
 
-        internal IdentifierAuthority IdentifierAuthority
-        {
-            get
-            {
-                return _identifierAuthority;
-            }
-        }
+        internal IdentifierAuthority IdentifierAuthority => _identifierAuthority;
 
-        internal int SubAuthorityCount
-        {
-            get
-            {
-                return _subAuthorities.Length;
-            }
-        }
+        internal int SubAuthorityCount => _subAuthorities.Length;
```

Comment:
You changed some calls to this to just use the equivalent of _subAuthorities.Length.  Do other call sites remain, or can it be deleted?

Reply:
There are other call sites.